app/common/src/queryClient.ts

```diff
@@ -56,7 +56,7 @@ declare module '@tanstack/query-core' {
 /** Query Client type suitable for shared use in React and Vue. */
 export type QueryClient = vueQuery.QueryClient
 
-const DEFAULT_QUERY_STALE_TIME_MS = 2 * 60 * 1000
+const DEFAULT_QUERY_STALE_TIME_MS = Infinity
```

Comment:
Does it mean that we never update queried data? It may be a problem for our cloud browser, which won't be refreshed after putting a new file to cloud directory.

Reply:
It's a temporary solution to avoid ton of setTimeout calls during subscribe

Instead of setting stale time we now have a query that mark all queries as stale each 2 minutes.

Worth noting that marking query as stale doesn't mean that it will be refetched immidiately.

Also if we use the same cache and queries across, we'll have a consistent state between vue and react

Reply:
> Instead of setting stale time we now have a query that mark all queries as stale each 2 minutes.

Then it's a good idea to leave comment about it here.

Reply:
We'll remove it once react-query landed a fix (they already opened a PR)